install_config/router/f5_router.adoc

```diff
@@ -129,29 +127,29 @@ $ oc adm policy add-scc-to-user privileged -z router
 ----
 ====
 
-Deploy the F5 router with the `oc adm router` command, but provide additional
+Deploy the F5 router plug-in with the `oc adm router` command, but provide additional
 flags (or environment variables) specifying the following parameters for the *F5
-BIG-IP®* host:
+BIG-IP* host:
 
 [[f5-router-flags]]
 [cols="1,4"]
 |===
 |Flag |Description
 
 |`--type=f5-router`
-|Specifies that an F5 router should be launched (the default `--type` is
+|Specifies that an F5 router plug-in should be launched (the default `--type` is
```

Comment:
|Specifies to launch an F5 router plug-in instead of the default *haproxy-router*.

Reply:
agree